model/fn-execution/src/main/proto/beam_provision_api.proto

```diff
@@ -71,6 +72,11 @@ message ProvisionInfo {
     // (required) The artifact retrieval token produced by
     // ArtifactStagingService.CommitManifestResponse.
     string retrieval_token = 6;
+
+    // (optional) The endpoint for runner to use for hosting the worker status
```

Comment:
please update comment to:
```
(optional) The endpoint that the runner is hosting for the SDK to submit status reports to during pipeline execution. This field will only be populated if the runner supports SDK status reports. For more details see {link to design doc}.
```

Reply:
Thanks Luke for review, I've applied the change.